src/palace/manager/core/opds2_import.py

```diff
@@ -741,6 +683,24 @@ def _extract_identifier(
         """
         return self.parse_identifier(publication.metadata.identifier)  # type: ignore[no-any-return]
 
+    @classmethod
+    def _extract_availability(
+        cls, availability: opds2_ast.OPDS2AvailabilityInformation | None
+    ) -> bool:
+        """Extract the publication's availability from its availability information.
+
```

Comment:
Is our stance here that, if an `availability` is present, but we are past the `until` date/time, that we simply act as if the `availability` prop is not present and then just use the default availability behavior?

Might be worth a comment in the docstring here. 

Also might be worthwhile to to have a `DEFAULT_AVAILABILITY` (`True` for our current behavior) and distinguish between an active availability property and the default more explicitly.

Reply:
Yes, that is what my assumption is here. I'll add a doc comment to make it more explicit. I don't think we have enough information for any other behaviour, since we don't have any indication of what the future state would be.

Reply:
Yeah, that makes sense.

Reply:
I added a more detailed comment on the `_extract_availability` function.